doc/contributing/contributing_guidelines.md

```diff
@@ -188,7 +189,66 @@ Work in progress.
 [Top](#top)
 
 ##<a name="section9"></a>Documentation
-Work in progress.
+###<a name="section9.1"></a>Repository documentation
```

Comment:
Every Cygnus Agent MUST include a documentation folder as per the following rules: